nautobot/dcim/models/power.py

```diff
@@ -98,7 +109,20 @@ class PowerFeed(PrimaryModel, PathEndpoint, CableTermination):
     An electrical circuit delivered from a PowerPanel.
     """
 
-    power_panel = models.ForeignKey(to="PowerPanel", on_delete=models.PROTECT, related_name="power_feeds")
+    power_panel = models.ForeignKey(
+        to="PowerPanel",
+        on_delete=models.PROTECT,
+        related_name="power_feeds",
+        help_text="Source panel that originates this power feed",
+    )
+    destination_panel = models.ForeignKey(
+        to="PowerPanel",
+        on_delete=models.PROTECT,
+        blank=True,
+        null=True,
+        related_name="feeders",
+        help_text="Destination panel that receives power from this feed",
+    )
```

Comment:
Please summarize the additions/changes to PowerFeed in a `.added`/`.changed` fragment(s) as well.